src/site/content/en/blog/text-fragments/index.md

```diff
@@ -0,0 +1,435 @@
+Notice the `hash` property with the value `#HTML1`.
+
+```js/3
+new URL('https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#HTML1');
+/* Creates a new `URL` object
+URL {
+  hash: "#HTML1"
+  host: "blog.chromium.org"
+  hostname: "blog.chromium.org"
+  href: "https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#HTML1"
+  origin: "https://blog.chromium.org"
+  password: ""
+  pathname: "/2019/12/chrome-80-content-indexing-es-modules.html"
+  port: ""
+  protocol: "https:"
+  search: ""
+  searchParams: URLSearchParams {}
+  username: ""
+}
+*/
+```
+
+The fact though that I had to open the Developer Tools to find the `id`
+of an element speaks volumes about the probability this particular section of the page
+was meant to be linked to by the author of the blog post.
+
+What if I want to link to something without an `id`?
+Say I want to link to the *ECMAScript Modules in Web Workers* heading.
+As you can see in the screenshot below, the `<h1>` in question does not have an `id` attribute,
+meaning there is no way I can link to this heading.
+This is the problem that Text Fragments solve 🎉.
+
+<figure class="w-figure">
+  <img src="id-missing.png" alt="" class="w-screenshot" width="400">
+  <figcaption class="w-figcaption">Dev Tools showing a heading without an <code>`id`</code>.</figcaption>
+</figure>
+
+## Text Fragments
+
+The [Text Fragments](https://wicg.github.io/ScrollToTextFragment/) proposal adds support
+for specifying a text snippet in the URL hash.
+When navigating to a URL with such a text fragment, the user agent can emphasize
+and/or bring it to the user's attention.
+
+### `textStart`
+
+In its simplest form, the syntax of Text Fragments is as follows:
+The hash symbol `#` followed by `:~:text=` and finally `textStart`, which represents the
+[percent-encoded](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)
+text I want to link to.
+
+```bash
+#:~:text=textStart
+```
+
+For example, say that I want to link to the
+*ECMAScript Modules in Web Workers* heading in the [blog post announcing features in Chrome 80](https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html), the URL in this case would be:
+
+<a href="https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html#:~:text=ECMAScript%20Modules%20in%20Web%20Workers"><code>https://blog.chromium.org/2019/12/chrome-80-content-indexing-es-modules.html<mark class="highlight-line highlight-line-active">#:~:text=ECMAScript%20Modules%20in%20Web%20Workers</mark></code></a>
+
+The text fragment is emphasized <mark class="highlight-line highlight-line-active">like this</mark>.
+If you click the link in a supporting browser like Chrome, the text fragment is
+highlighted and scrolls into view:
+
+<figure class="w-figure">
+  <img src="syntax-simple.png" alt="" class="w-screenshot" width="400">
```

Comment:
Also small but perhaps this one was intentional

![image](https://user-images.githubusercontent.com/4713486/81129043-e186de00-8ef7-11ea-961e-b83ac16e3d77.png)

Reply:
Intentional indeed. Just the heading is important.